README.md

```diff
@@ -63,22 +63,19 @@ Items:
 >>>  ts = dfs.to_dataframe('simple.dfs0')
 ```
 
-### Create simple timeseries
+### Write simple timeseries
 ```python
->>>  from datetime import datetime, timedelta
+>>>  from datetime import datetime
 >>>  import numpy as np
 >>>  from mikeio import Dfs0
 >>>  data = []
 >>>  d = np.random.random([100])
 >>>  data.append(d)
 >>>  dfs = Dfs0()
->>>  dfs.create(filename='simple.dfs0',
->>>            data=data,
->>>            start_time=datetime(2017, 1, 1),
->>>            dt=60)
+>>>  dfs.write('simple.dfs0', data, start_time=datetime(2017, 1, 1), dt=60)
```

Comment:
maybe make the example more concise by:


    >>>  data = [np.random.random([100])]
    >>>  dfs = Dfs0()
    >>>  dfs.write('simple.dfs0', data, start_time=datetime(2017, 1, 1), dt=60)
?

Reply:
Good idea, done